pkg/operator/reports_test.go

```diff
@@ -5,19 +5,23 @@ import (
 	"time"
 
 	"github.com/operator-framework/operator-metering/pkg/apis/metering/v1alpha1"
+	metering "github.com/operator-framework/operator-metering/pkg/apis/metering/v1alpha1"
+	"github.com/operator-framework/operator-metering/test/testhelpers"
+	"github.com/sirupsen/logrus"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
Generally we use `metav1` as the package alias since `v1` usually refers to the `k8s.io/api/core/v1` package.